test-three-nathanpeter/menu/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
Noticed that you included the tests from the create react app. Is there a reason why you didnt modify to pass your changes?

Reply:
sorry that was just a mistake didnt take it out